docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -65,6 +66,55 @@ Applications that target versions of .NET Framework starting with 4.7.1 can disa
+- Addressed an issue which caused screen readers to count hidden columns when announcing the column count in a <xref:System.Windows.Forms.DataGridView> control.
+- Addressed an issue cuaing the <xref:System.Windows.Forms.DataGridView> to ignore the font settings set in the `DataGridviewCellStyle` if the underlying form has a `Font` property that differs from the `DefaultFont`
+- Updated the `AccessibleName` property of the <xref:System.Windows.Forms.DataGridView> control's internal scroll bars to remove the text "ScrollBar".
+- Fixed the color of a `DataGridViewLinkCell` when the cell is selected.
+- Improved keyboard navigation in a <xref:System.Windows.Forms.ToolStrip> when the element is a `ToolStripComboBox` type.
+- Fixed an issue with the <xref:System.Windows.Forms.ComboBox> which could cause application crash under Accessibility Insights for Windows.
+- Updated the border color for the <xref:System.Windows.Forms.Button> to have more contrast in the default colors.
+- Ensured that there was a bounding rectangle reported to assitive technology for the <xref:System.Windows.Forms.ToolStripSeparator>
+- Enabled assistive technology tools to access the `ControlBox` buttons of a maximized MDI Child form.
+- Updated the background color of the <xref:System.Windows.Forms.ToolStripButton> in High Contrast mode.
+- <xref:System.Windows.Forms.DomainUpDown> control `AccessibleName` property has a new default value of an empty string. This will encourage developers to create a meanignful name rather than accepting a default of "Spinner"
+- Ensured that the UIA Hierarchy tree for a <xref:System.Windows.Forms.PropertyGrid> control when a complex entry, like `Font` is expaneded, and updated properly when the entry is then collapsed and no longer visible.
+- Updated the `AccessibleName` property of the Print button in the <xref:System.Windows.Forms.PrintPreveiwDialog> from "Print Button" to "Print" to avoid redundancy when screen readers announce the control and its type.
+- <xref:System.Windows.Forms.PropertyGrid> categories now have a localized control type of "PropertyGrid category"
+- Fixed an issue with custom <xref:System.Windows.Forms.DataGridView> where there was no ControlType or LocalizedControlType provided publicly for custom <xref:System.Windows.Forms.DataGridViewCell> elements.
+- Removed an empty UIA list element when a <xref:System.Windows.Forms.PropertyGridCell> of type `ComboBox` is closed and no longer visible.
+
+
+
+
+<a name="wpf48"></a>
+### Windows Presentation Foundation (WPF)
+####Accessible Tooltip handling improvements
+In this release WPF improved the experience by ensuring that a tooltip in the current window can easily be dismissed by using the ESC key, the CTRL key (by itself), or by the combination Ctrl+Shift+F10. The scope of the Escape key was reduced in this release to apply only to the current window, when previously it would have been any open tooltip in the application.
```

Comment:
```suggestion
In this release WPF improved the experience by ensuring that a tooltip in the current window can easily be dismissed by using the ESC key, the CTRL key (by itself), or by the combination Ctrl+Shift+F10. The scope of the ESC key was reduced in this release to apply only to the current window, when previously it would have been any open tooltip in the application.
```